src/Microsoft.ML.DnnAnalyzer/Microsoft.ML.DnnAnalyzer/Microsoft.ML.DnnAnalyzer.csproj

```diff
@@ -4,7 +4,6 @@
     <OutputType>Exe</OutputType>
     <TargetFramework>netcoreapp2.0</TargetFramework>
```

Comment:
>netcoreapp2.0 [](start = 21, length = 13)

Comment from @eerhardt from previous PR: 
>This should be netcoreapp2.1. Note that netcoreapp and netstandard are different things. We want netcoreapp2.1 for any and all executables and tests. We want netstandard2.0 for any libraries.

Reply:
I'd prefer we just nuke this thing altogether and recast it as an `ICommand`. It should have been one anyway.